package.json

```diff
@@ -1,153 +1,154 @@
-    "@types/koa-static": "^4.0.1",
-    "@types/mocha": "^8.2.0",
-    "@types/ms": "^0.7.31",
-    "@types/node": "^10.17.35",
-    "@types/request": "^2.48.5",
-    "@types/request-promise": "^4.1.47",
-    "@types/semver": "^7.3.4",
-    "@types/ws": "^7.4.0",
-    "@types/yargs": "^16.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.15.0",
-    "@typescript-eslint/parser": "^4.15.0",
-    "chai": "^4.3.0",
-    "chai-as-promised": "^7.1.1",
-    "chalk": "^4.1.0",
-    "clean-webpack-plugin": "^3.0.0",
-    "concat-stream": "^2.0.0",
-    "eslint": "^7.19.0",
-    "eslint-config-prettier": "^7.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-prefer-arrow": "^1.2.3",
-    "husky": "^5.1.1",
-    "lint-staged": "^10.5.4",
-    "mocha": "^8.2.1",
-    "nodemon": "^2.0.7",
-    "nyc": "^15.1.0",
-    "pkg": "^4.4.9",
-    "prettier": "^2.2.1",
-    "semantic-release": "^17.3.8",
-    "terser-webpack-plugin": "^5.1.1",
-    "ts-loader": "^8.0.16",
-    "ts-node": "^9.1.1",
-    "tsconfig-paths-webpack-plugin": "^3.3.0",
-    "typescript": "^4.1.3",
-    "webpack": "^5.21.2",
-    "webpack-cli": "^4.5.0",
-    "webpack-node-externals": "^2.5.2"
-  },
-  "files": [
-    "dist"
-  ],
-  "homepage": "https://github.com/NeuraLegion/nexploit-cli#readme",
-  "husky": {
-    "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-commit": "lint-staged",
-      "post-commit": "git update-index --again"
-    }
-  },
-  "keywords": [
-    "nexploit",
-    "cli",
-    "api",
-    "nexploit-cli",
-    "cyber-security",
-    "security",
-    "har",
-    "oas",
-    "crawler",
-    "nexmock"
-  ],
-  "license": "MIT",
-  "lint-staged": {
-    "*.ts": [
-      "eslint --fix",
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "scripts": {
-    "lint": "eslint -c .eslintrc.js src",
-    "format": "prettier --check src",
-    "test:unit": "mocha -r ts-node/register -r tsconfig-paths/register src/**/*.spec.ts",
-    "test:e2e": "mocha -r ts-node/register -r tsconfig-paths/register tests/**/*.spec.ts",
-    "build": "webpack --config webpack.config.js --mode=production",
-    "semantic-release": "semantic-release",
-    "pack:win": "pkg -o bin/win/nexploit-cli.exe -t node14-windows-x64 .",
-    "pack:mac": "pkg -o bin/mac/nexploit-cli -t node14-mac-x64 .",
-    "pack:linux": "pkg -o bin/linux/nexploit-cli -t node14-linux-x64 ."
-  },
-  "pkg": {
-    "assets": [
-      "dist/public/**/*"
-    ]
-  }
-}
+{
+  "name": "@neuralegion/nexploit-cli",
+  "version": "0.0.0",
+  "private": false,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/NeuraLegion/nexploit-cli.git"
+  },
+  "author": "Artem Derevnjuk <artem.derevnjuk@neuralegion.com>",
+  "bin": {
+    "nexploit-cli": "./dist/index.js"
+  },
+  "bugs": {
+    "url": "https://github.com/NeuraLegion/nexploit-cli/issues"
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },

```

Comment:
use `npm run format -- --write && npm run lint -- --fix` to avoid such formatting issues